src/ui/components/atoms/label-icon/ButtonLabel.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './styles.scss';
+import type { IProps } from './types/IProps';
+
+function ButtonLabel({ children, variant = 'primary', icon }: IProps) {
+  return (
+    <div className="container">
```

Comment:
please don´t use container like a name use another for example label-icon-container